File: backend/Domain/Entities/archive_file_tags.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Entities
{
    public class archive_file_tags
    {
        public int id { get; set; }
		public DateTime? created_at { get; set; }
		public DateTime? updated_at { get; set; }
		public int? created_by { get; set; }
		public int? updated_by { get; set; }
		public int? file_id { get; set; }
		public int? tag_id { get; set; }
		
    }
}

File: frontend/src/constants/archive_file_tags.ts
import { Dayjs } from "dayjs";

export type archive_file_tags = {
  
  id: number;
  created_at: Dayjs;
  updated_at: Dayjs;
  created_by: number;
  updated_by: number;
  file_id: number;
  tag_id: number;
};
